examples/jenkins/README.md

```diff
@@ -9,83 +9,67 @@ the deployment works, and then tag the test version into production.
     
-4. Retrieve the ip and port of the jenkins service that was just created:
-   
-        $ oc get svc jenkins
+1. View/Manage Jenkins
+
+    If you have a router running (`oc cluster up` provides one), run:
 
-    Note the ip and port of the Jenkins service reported by the second command, you will need it later.
+        $ oc get route
 
-5. Create the sample application configuration
+    and access the host for the Jenkins route.
 
-        $ oc new-app application-template.json
+    If you do not have a router or your host system does not support xip.io name resolution, you can access jenkins directly via the service ip.  Determine the jenkins service ip ("oc get svc") and go to it in your browser on port 80.  Do not confuse it with the jenkins-jnlp service.
 
-    Note the generated password (JENKINS_PASSWORD) reported in the new-app output, you will need it to login to Jenkins.  If you lose this value
-    you can retrieve it by looking at the environment variable values defined on the jenkins deployment configuration in the web console or by
-    running:
+    **Note**: The OpenShift Login plugin by default manages authentication into any Jenkins instance running in OpenShift.  When this is the case, and you do intend to access Jenkins via the Service IP and not the Route, then you will need to annotate the Jenkins service account with a redirect URL so that the OAuth server's whitelist is updated and allow the login to Jenkins to complete. 
 
-        $ oc env dc/jenkins --list | grep JENKINS_PASSWORD
+        $ oc annotate sa/jenkins serviceaccounts.openshift.io/oauth-redirecturi.1=http://<jenkins_service_ip:jenkins_service_port>/securityRealm/finishLogin --overwrite
  
-6. Open the Jenkins service ip:port from step 4 in your browser.  Once it is available, login using username `admin` and the password from the previous step.
-   
-7. Select the the `OpenShift Sample` job and click `Configure`.  You'll see a series of Jenkins build steps defined.  These build steps are from the Jenkins plugin for V3 Openshift.  Read about the [OpenShift Jenkins plugin](https://github.com/openshift/jenkins-plugin) for details on the various functionality provided.  The default values for each of the various build steps listed for the sample job should work as is.  You can save your changes to the job, click `Build` and skip to step 11.
+    Login with the user name you supplied to `oc login` and any non-empty password.
+
+1. Create the sample application configuration
+
+        $ oc new-app -f https://raw.githubusercontent.com/openshift/origin/master/examples/jenkins/application-template.json
```

Comment:
I think we should move this step above the "view/manage jenkins" step.  that way they get all the CLI context steps out of the way, and then switch into the Jenkins console context.